app/controllers/variables_controller.rb

```diff
@@ -22,6 +22,13 @@ def edit
     end
   end
 
+  def edit
```

Comment:
If you expand or view the whole file, it looks like this PR is now inserting a second `edit` method